docs/nidcpower/class.rst

```diff
@@ -718,14 +714,10 @@ fetch_multiple
             .. note:: This method is not supported on all devices. Refer to `Supported Methods by Device <REPLACE_DRIVER_SPECIFIC_URL_2(nidcpowercref.chm, supportedfunctions)>`__ for more information about supported devices.
 
 
-            .. tip:: This method requires repeated capabilities (channels). If called directly on the
+            .. tip:: This method requires repeated capabilities. If called directly on the
                 nidcpower.Session object, then the method will use all repeated capabilities in the session.
                 You can specify a subset of repeated capabilities using the Python index notation on an
-                nidcpower.Session repeated capabilities container, and calling this method on the result.:
-
-                .. code:: python
-
-                    session.channels[0,1].fetch_multiple(count, timeout=datetime.timedelta(seconds=1.0))
+                nidcpower.Session repeated capabilities container, and calling this method on the result.
```

Comment:
Will a customer know what a "nidcpower.Session repeated capabilities container" is and how would they even know what "nidcpower.Session repeated capabilities containers" are available?

At least the sometimes misleading example worked 90% of the time.

Reply:
Moved repeated capabilities documentation from the middle of class.rst to a new file that will get more visibility. See #1317 

We can work on how to word the documentation there as well as see about adding package specific content.